test/events.test.ts

```diff
@@ -1,9 +1,10 @@
 import { TestOctokit } from "./octokit";
```

Comment:
Having problems with the test coverage. Apparently the `%Func` coverage for `index.ts` decreased from `100` to `80` and I don't find which is it. I will check it later but if you have a clue it will be very welcome!